README.md

```diff
@@ -33,27 +33,28 @@ If you have already an ERC20 token for ICO, you can skip this.
 
 ![img_5.png](imgs/img_5.png)
 
-- Search and select ICO module
+- Search and select [ICO (Public Sale) module](https://app.bunzz.dev/module-templates/c922eb26-35b6-4c88-a667-a51ceba650f3)
 
-![img_7.png](imgs/img_7.png)
+![img_30.png](imgs/img_30.png)
 
-![img_8.png](imgs/img_8.png)
+![img_31.png](imgs/img_31.png)
 
 - Input token name and symbol. (If you didn't select ERC20 module, you don't need to add the token name and symbol).
+- Input start time and end time in timestamp type. (endTime > startTime >= currentTime)
 
-![img_9.png](imgs/img_9.png)
+![img_32.png](imgs/img_32.png)
```

Comment:
Please make a screenshot here and on this [image](https://github.com/lastrust/boilerplate-for-ICO-Staking/blob/base-app/imgs/img_17.png) with value examples🙏
It will help users understand, which values must be in the field.